src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -299,6 +299,17 @@ report_status(xrt_core::device_collection& deviceCollection, boost::property_tre
 
 }
 
+static void 
+isSameShellOrSC(DSAInfo& candidate, DSAInfo& current, bool *same_dsa, bool *same_bmc)
```

Comment:
Initialize return value to a know known value.

Reply:
I'm not sure what you mean by return a known value. We are getting the return values in the 2 booleans, so the method returns void